src/editor/Core/ConfigurationDefaults.re

```diff
@@ -30,7 +30,8 @@ let getDefaultConfigString = configName =>
   "workbench.editor.showTabs": true,
   "workbench.iconTheme": "vs-seti",
   "workbench.sideBar.visible": true,
-  "workbench.statusBar.visible": true
+  "workbench.statusBar.visible": true,
+  "workbench.tree.indent": 2,
```

Comment:
It looks like the configuration-defaults test was failing - it didn't like this trailing comma. I went ahead and removed it 👍